appinventor/appengine/src/com/google/appinventor/server/DownloadServlet.java

```diff
@@ -237,22 +266,40 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOExc
       out.write(message.getBytes());
       out.close();
       return;
+    } catch (NoSuchAlgorithmException e) {
+      throw CrashReport.createAndLogError(LOG, req, "user=" + userId, e);
     }
 
-    String fileName = downloadableFile.getFileName();
-    byte[] content = downloadableFile.getContent();
+    resp.setStatus(status_code);
 
-    // Set http response information
-    resp.setStatus(HttpServletResponse.SC_OK);
-    resp.setHeader(
+    if (status_code == HttpServletResponse.SC_OK) {
+      LOG.info("Sending File!");
```

Comment:
I wonder if we should reduce the log level or put this behind a debug flag. This will result in a lot of log messages in production.

Reply:
I'll take care of this.

Reply:
I changed the LOG.info to LOG.fine. This will let us see messages by just editing logging.properties instead of re-compiling.